site/en/tutorials/structured_data/time_series.ipynb

```diff
@@ -611,13 +614,13 @@
       "source": [
         "Depending on the task and type of model you may want to generate a variety of data windows. Here are some examples:\n",
         "\n",
-        "1. For example, to make a single prediction 24h into the future, given 24h of history you might define a window like this:\n",
+        "1. For example, to make a single prediction 24 hours into the future, given 24 hours of history, you might define a window like this:\n",
         "\n",
-        "  ![One prediction 24h into the future.](images/raw_window_24h.png)\n",
+        "  ![One prediction 24 hours into the future.](images/raw_window_24h.png)\n",
         "\n",
-        "2. A model that makes a prediction 1h into the future, given 6h of history would need a window like this:\n",
+        "2. A model that makes a prediction 1 hour into the future, given 6 hours of history, would need a window like this:\n",
```

Comment:
```suggestion
        "2. A model that makes a prediction one hour into the future, given 6 hours of history, would need a window like this:\n",
```

Style: Use words for numbers zero through nine. Then numbers for 10 and up
https://developers.google.com/style/numbers?

Reply:
(here and elsewhere in this PR)

Reply:
Done. (Left the plot title unchanged with `1` and `6`.)